src/lib.rs

```diff
@@ -120,3 +120,11 @@ cfg_rt! {
 
 mod task;
 pub use task::{Instrumented, TaskMetrics, TaskMonitor};
+
+#[cfg(unix)]
+pub mod lrtd;
+#[cfg(unix)]
+pub use lrtd::{
```

Comment:
this is a very minor style nit, but i'm not a _huge_ fan of the acronym module name, since it's not immediately obvious to the reader what "LRTD" stands for. it's a bit wordier, but I might call it `long_running_tasks`, and shorten the name of the struct `LongRunningTaskDetector` to `long_running_tasks::Detector` or something?

i'm open to being convinced otherwise about this, though!

Reply:
I agree, let me think about a better name.

Reply:
@hawkw  updated the name to detectors::LongRunningTaskDetector should be better, hope it's good enough.